rlptxn/constraints.lisp

```diff
@@ -858,7 +864,9 @@
                                                 (begin (vanishes! (+ PHASE_END
                                                                      (next LT)
                                                                      (- 1 (next LX))
-                                                                     (- 1 (next IS_PREFIX)))))))))))
+                                                                     (- 1 (next IS_PREFIX))))
+                                                       (eq! (^ (w-minus-two-beta-minus-protected-base-V) 2)
+                                                            (w-minus-two-beta-minus-protected-base-V)))))))))
```

Comment:
We could just use `(is-binary (w-minus-two-beta-minus-protected-base-V))`, no ?

Reply:
yes, but I plan to do it globally as I have this perf issue in lot of space in RLP_* module : https://github.com/Consensys/linea-constraints/issues/396

Reply:
hum ... c'est pas faux